doc/attributes.md

```diff
@@ -0,0 +1,51 @@
+# User And Group Attributes
+
+This document describes the attributes that can be indicated for users and groups, either through the YaST UI of the *users* client or through AutoYaST. Each attribute should have a counterpart in the new *Y2Users* code. This correspondence helps to translate the users data structure from *Yast::Users* perl module to the new code.
+
+## User
+
+The YaST UI for creating and editing a user offers the following attributes:
+
+| UI Field | Y2Users attr | Only when adding | Only when editing | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| full name | gecos | | | |
+| name | name | | | |
+| password | password | | | |
+| system mail | | | | |
+| disable login | * password starting by !$ | | | |
```

Comment:
More exactly, password starting with `!`. See [usermod(8) man page](https://linux.die.net/man/8/usermod)

> ...
> 
> **-L, --lock**
    Lock a user's password. This puts a '!' in front of the encrypted password, effectively disabling the password. You can't use this option with -p or -U.
> 
>     Note: if you wish to lock the account (not only access with a password), you should also set the EXPIRE_DATE to 1. 
> 
> ...
> 
> **-U, --unlock**
    Unlock a user's password. This removes the '!' in front of the encrypted password. You can't use this option with -p or -L.
> 
>    Note: if you wish to unlock the account (not only access with a password), you should also set the EXPIRE_DATE (for example to 99999, or to the EXPIRE value from /etc/default/useradd).